src/generate_letters.ml

```diff
@@ -0,0 +1,33 @@
+(** letter_gen is a list containing 7 random chars at any time.
+    To be used in a game of scrabble. If the char has been used in the game,
+    there will be an '\000' in its place.
+    Requires each char in the list to be an alphabetical letter or '\000'. *)
+type letter_gen = char list
+
+(** [random_let] is a randomly generated char from a-z.
+    random_let : char*)
+let random_let : char = 
```

Comment:
It seems like `random_let` is a char variable (`char`) and not a function (`unit -> char`), which means that every time you use `random_let` it will resolve to the same char.